crashsimilarity/utils.py

```diff
@@ -48,6 +57,34 @@ def process(stream, take_top_funcs=None):
                 yield (processed, data['signature'].lower())
 
 
+def utc_today():
```

Comment:
Can you move this function and the following one to the top of the file, so the diff is smaller